libraries/core/src/main/java/com/paritytrading/philadelphia/FIXValue.java

```diff
@@ -85,6 +85,32 @@ public void set(FIXValue value) {
         System.arraycopy(value.bytes, 0, bytes, 0, length + 1);
     }
 
+    /**
+     * Get the value as a boolean.
+     *
+     * @return the value as a boolean
+     * @throws FIXValueFormatException if the value is not a boolean
+     */
+    public boolean asBoolean() {
+        if (length != 1)
```

Comment:
Is this a sufficient check that it's not a boolean? `notBoolean` and `notChar` are less meaningful than the other sanity checks (`notDigit` etc). Just thinking how much such sanity checks add.

Reply:
Sorry github was failing 😆.

Reply:
Good point on the general utility of these checks. 👍 

This one is modelled based on the `asChar` implementation. I added it, because we generally do check the length of the value. However, it would be good to think how much value these checks add altogether.